src/Arcus.Templates.Tests.Integration/WebApi/Swagger/v1/SwaggerDocExamplesTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Templates.Tests.Integration.WebApi.Swagger.v1
+{
+    [Collection(TestCollections.Integration)]
+    [Trait("Category", TestTraits.Integration)]
+    public class SwaggerDocExamplesTests
```

Comment:
Congrats on using the integration test API functionality! Was it self explanatory, or did you have to look hard to find what you're looking for?

Reply:
I went a little bit through the code and could quickly find what I was looking for.